src/Models/DatabaseNotification.php

```diff
@@ -94,4 +95,10 @@ public function hasAction(): bool
     {
         return (bool) data_get($this->data, 'action');
     }
+
+    public function excerpt(?int $length = null): string
+    {
+        /* @phpstan-ignore-next-line  */
```

Comment:
what does it ignore?

Reply:
The `strip_tags`, $this->content() can eventually be null, but strip_tags doesn't like that

Reply:
shouldn't we handle that then in case it can be null 🤔

Reply:
Will have a check now

Reply:
After looking at it, I think the ignore is fine. `strip_tags` works fine with a `null` string and doesn't throw any exceptions, it will just returns an empty `string` such as `""`. Trying to handle it differently doesn't make a lot of sense either because the `content()` method is defined to be eventually nullable.